<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlertSetting</class>
 <widget class="QWidget" name="AlertSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>681</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>定时提醒设置</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>日期</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>标题</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>内容</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="ListWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>661</width>
      <height>181</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>日程时间</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>日程主题</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>日程内容</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="TopicEdit">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>60</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="DetailEdit">
    <property name="geometry">
     <rect>
      <x>343</x>
      <y>60</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>详细内容</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>61</y>
     <width>71</width>
     <height>151</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QPushButton" name="AddButton">
    <property name="text">
     <string>增加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ModifyButton">
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteButton">
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CleanButton">
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>651</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
